test/integrations/destinations/linkedin_ads/processor/configLevelFeaturesTestData.ts

```diff
@@ -0,0 +1,223 @@
+import {
+  generateMetadata,
+  generateTrackPayload,
+  overrideDestination,
+  transformResultBuilder,
+} from '../../../testUtils';
+import { Destination } from '../../../../../src/types';
+import { ProcessorTestData } from '../../../testTypes';
+
+const commonDestination: Destination = {
+  ID: '12335',
+  Name: 'sample-destination',
+  DestinationDefinition: {
+    ID: '123',
+    Name: 'linkedin_ads',
+    DisplayName: 'LinkedIn Ads',
+    Config: {
+      cdkV2Enabled: true,
+    },
+  },
+  WorkspaceID: '123',
+  Transformations: [],
+  Config: {
+    hashData: true,
+    conversionMapping: [
+      {
+        from: 'ABC Searched',
+        to: '1234567',
+      },
+      {
+        from: 'spin_result',
+        to: '23456',
+      },
+      {
+        from: 'ABC Searched',
+        to: '34567',
+      },
+    ],
+    oneTrustCookieCategories: [
+      {
+        oneTrustCookieCategory: 'Marketing',
+      },
+    ],
+  },
+  Enabled: true,
+};
+
+const commonUserTraits = {
+  email: 'abc@gmail.com',
+  anonymousId: 'c82cbdff-e5be-4009-ac78-cdeea09ab4b1',
+  event_id: '12345',
+};
+
+const commonUserProperties = {
+  price: 400,
+  additional_bet_index: 0,
+  eventId: '12345',
+};
+
+const commonTimestamp = new Date('2023-10-14');
+
+export const configLevelFeaturesTestData: ProcessorTestData[] = [
+  {
+    id: 'linkedin_ads-config-test-1',
+    name: 'linkedin_ads',
+    description: 'Track call : hashData is set to false and no deduplication key is provided',
+    scenario: 'Business',
+    successCriteria: 'email provided will not be hashed and eventId will be mapped from messageId',
```

Comment:
I see 2 test cases covering the scenario of hashing config enabled or not. But I believe there are other configs

Conversion mapping we should also test that.